### PR TITLE
refactored to mina_commands, mina_plugins, mina_transition, mina_version

### PR DESCRIPTION
**Pull request by Kida007**
This PR: 
- refactored to mina_commands, mina_plugins, mina_transition, mina_version 

Tested:
- make build

Checklist:

- [ ] Document code purpose, how to use it
  - Mention expected invariants, implicit constraints
- [ ] Tests were added for the new behavior
  - Document test purpose, significance of failures
  - Test names should reflect their purpose
- [ ] All tests pass (CI will check this if you didn't)
- [ ] Serialized types are in stable-versioned modules
- [ ] Does this close issues? List them:

### CODE REVIEW

**Review by bkase — APPROVED** [review]

**Comment by bkase** [comment]
!ci-build-me

File: src/lib/mina_commands/mina_commands.ml [hunk 1]
```diff
@@ -75,7 +75,7 @@ let replace_block_production_keys keys pks =
   kps |> List.map ~f:snd
 
 let setup_and_submit_user_command t (user_command_input : User_command_input.t)
-    =
+  =
```
Comment (mrmr1993) [inline src/lib/mina_commands/mina_commands.ml:78]: According to CI, this file needs to be reformatted:
```
File: ./src/lib/mina_commands/mina_commands.ml has needs to be ocamlformat-ed
```
Can you run `make reformat` and update the PR with the formatting changes it generates?

Comment (Kida007) [reply src/lib/mina_commands/mina_commands.ml:78]: done . thanks for pointing out.

**Comment by mrmr1993** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

**Pull request merged**